lib/galaxy/datatypes/converters/vcf_to_vcf_bgzip.py

```diff
@@ -8,8 +8,7 @@
 usage: %prog in_file out_file
 """
 import optparse
-
-from pysam import ctabix
```

Comment:
I think you should keep the blank line, optparse is part of the standard library, while pysam is not.

Reply:
Yep, agreed, I'll tweak it in a sec.